db/system_keyspace.cc

```diff
@@ -1577,7 +1577,10 @@ template <typename T>
 future<> set_scylla_local_param_as(const sstring& key, const T& value) {
     sstring req = format("UPDATE system.{} SET value = ? WHERE key = ?", system_keyspace::SCYLLA_LOCAL);
     auto type = data_type_for<T>();
-    return qctx->execute_cql(req, type->to_string_impl(data_value(value)), key).discard_result();
+    co_await qctx->execute_cql(req, type->to_string_impl(data_value(value)), key).discard_result();
+    co_await smp::invoke_on_all([] () -> future<> {
```

Comment:
Let's have a comment explaining this flush so someone doesn't try to remove it in the future (or so we can remove it in ~2 years).